index.bs

```diff
@@ -2893,7 +2893,7 @@ When registering a new credential, represented by a {{AuthenticatorAttestationRe
     Note: Since all extensions are OPTIONAL for both the [=client=] and the [=authenticator=], the [=[RP]=] MUST be prepared to
     handle cases where none or not all of the requested extensions were acted upon.
 
-1. Determine the attestation statement format by performing an USASCII case-sensitive match on |fmt| against the set of
+1. Determine the attestation statement format by performing a USASCII case-sensitive match on |fmt| against the set of
```

Comment:
FYI, "a USASCII" is correct because of phonetics.  "USASCII" sounds like starts with a "y". ;-)

Reply:
Great, that's what I thought as well. :)